### PR TITLE
Migration trial: Update migration trial step design (version 2)

### LINKED ISSUE #80499: UI: Update migration trial step (version 2)
Image

### PR DESCRIPTION
**Pull request by bogiii**
Closes #80499

## Proposed Changes

* Update Migration step design

## Testing Instructions

* Go to `/setup/import-focused/migrationTrial?siteSlug={SITE_SLUG}&from={JN_SITE}&option=everything`
* Check if there is a new version of the migration trial step

Screenshot 2023-08-14 at 00 07 23

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/migration-trial/trial-plan.tsx [hunk 1]
```diff
@@ -111,42 +111,35 @@ const TrialPlan = function ( props: Props ) {
...
 							) ) }
 						</ul>
 					</div>
 				</div>
 
+				<div className="trial-plan--details-limitation">
+					<img src={ clockIcon } alt={ __( 'Limit' ) } />
+					<strong>{ __( 'Trial limitations' ) }</strong>
+					<p>
+						<small>No custom domains, 100 subscribers, no SSH or SFTP access</small>
```
Comment (ouikhuan) [inline client/landing/stepper/declarative-flow/internals/steps-repository/migration-trial/trial-plan.tsx]: Should this be translatable string here?

**Review by ouikhuan — COMMENTED** [review]
In Firefox, the word breaks into two line, but this is just something I noticed, not a blocker here,
Screen Shot 2023-08-15 at 5 58 22 PM

File: client/landing/stepper/declarative-flow/internals/steps-repository/migration-trial/trial-plan.tsx (hunk 1 above)
Comment (bogiii) [reply client/landing/stepper/declarative-flow/internals/steps-repository/migration-trial/trial-plan.tsx]: For sure... good catch!

**Review by ouikhuan — APPROVED** [review]
LGTM 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8716205

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8716205).

**Pull request merged**